internal/controller/k0smotron.io/k0smotroncluster_controller.go

```diff
@@ -39,6 +41,8 @@ import (
 	km "github.com/k0sproject/k0smotron/api/k0smotron.io/v1beta1"
 )
 
+const serviceAccountName = "k0smotron"
```

Comment:
I'd maybe rename this to something like `k0smotron-hcp` just to say that it's NOT for k0rmotron itself